docs/DocumentationStyleGuide/style-tone.rst

```diff
@@ -0,0 +1,90 @@
+Style & Tone
+===========
+
+We aim to achieve a simple, easy-to-understand context for our technical
+and non-technical documentation. It aims to overlook the too-formal or
+robotic tone and give way to a friendly, inclusive and respectful tone
+of communication. It should call to action in a casual and approachable
+tone and should not sound pushy or dry.
+
+As a Technical Writer, if you are using an entertaining tone to help the
+readers better understand, do not go over the humour part. We generally
+assume that not every humour piece is universal and can be interpreted
+differently across different regions.
+
+A few guidelines are employed to understand the style and tone. These
+are entirely flexible and are not universal for every context that we
+share amongst ourselves.
+
+Approaches to be considered
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Use Active voice wherever possible.
+-  The style and tone should be friendly and come off as natural.
+-  Making use of transaction words is quite preferred.
+-  Simplify the documentation hierarchy for the readers to easily
+   follow.
+-  Technical terms, abbreviations, jargon should be documented early on
+   in the document.
+-  Brief, concise statements are preferred over wordy sentences.
+-  Documentation pieces should be proofread by a peer or a fellow
+   contributor.
+-  Gender-neutral pronouns must be used for overall inclusivity.
+
+Approaches to be not considered
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Using gender-specific and culture-specific terminologies and figures
+   of speech.
+-  Using biased, unconfirmed or pre-assumed information pieces.
+-  Using pretentious language to convince the reader of a fact or
```

Comment:
This is a good point, but it's not immediately clear what 'pretentious language' - maybe an accessible or humourous example would help?